lemur/manage.py

```diff
@@ -583,24 +651,18 @@ def publish_verisign_units():
 
 
 def main():
-    manager.add_command("start", LemurServer())
-    manager.add_command("runserver", Server(host="127.0.0.1", threaded=True))
```

Comment:
do we have any equivalent command, or did we need to remove entirely?

Reply:
Here is the current list of commands for the cli:

```
Commands:
  acme                    Handles all ACME related tasks
  certificate             Handles all certificate related tasks.
  create_config           Creates a new configuration file if one does...
  create_role             This command allows for the creation of a new...
  create_user             This command allows for the creation of a new...
  db                      Perform database migrations.
  dns_providers           Iterates through all DNS providers and sets DNS...
  init                    This command will bootstrap our database with...
  lock                    Encrypts a given path.
  notify                  Handles notification related tasks.
  pending_certs           Handles pending certificate related tasks.
  policy                  Handles all policy related tasks.
  publish_verisign_units  Simple function that queries verisign for API...
  report                  Reporting related tasks.
  reset_password          This command allows you to reset a user's...
  routes                  Show the routes for the app.
  run                     Run a development server.
  shell                   Run a shell in the app context.
  source                  Handles all source related tasks.
  start                   This is the main Lemur server, it runs the...
  unlock                  Decrypts all of the files in a given directory...
``` 
The following `ShowUrls, Clean, Server` are flask-script commands that have been removed and replaced by click and FlaskGroup. The `runserver` cmd has been replaced by a default `run` cmd,   the `show_urls` cmd has been replaced with `routes`, and currently there is no default replacement for the `clean` cmd.

The other commands have been reformatted to the click format. For example, the `add_command` for `create_user` was removed and replaced with the `click` format `@cli.command("create_user")` on Line 307.

Currently, the only command we are missing is the `clean` cmd, which was a default command from the flask-script commands.

Reply:
Looking to see if there is a replacement command or find the best way to implement the `clean` command in Flask2/Click.

Reply:
from looking at the source, it seems to just clean pyc and pyo files 
```
        for dirpath, dirnames, filenames in os.walk('.'):
            for filename in filenames:
                if filename.endswith('.pyc') or filename.endswith('.pyo'):
                    full_pathname = os.path.join(dirpath, filename)
                    print('Removing %s' % full_pathname)
                    os.remove(full_pathname)
```
I'm fine with leaving this functionality out and adding it back in if it causes a breakage. We should add a note to the unreleased section of the changelog with this changes though. e.g. "cli: The runserver cmd has been replaced by a default run cmd, the show_urls cmd has been replaced with routes, and currently there is no default replacement for the clean cmd." Also, if there are any other known breaking changes in this PR, that'd be a good place to call them out as well.

Reply:
sounds good. i agree, hopefully it won't cause any breakage 🤞. changelog has been updated.